### ISSUE TITLE
"error: NotReadableError: Could not start video source" when pushing by Android Pixel 4A5G of back camera with Chrome

### ISSUE THREAD

**Issue by iesv** [bluenviron/mediamtx#2698]
## Which version are you using?

v1.3.0

## Which operating system are you using?

- [ ] Linux amd64 standard
- [ ] Linux amd64 Docker
- [ ] Linux arm64 standard
- [ ] Linux arm64 Docker
- [ ] Linux arm7 standard
- [ ] Linux arm7 Docker
- [ ] Linux arm6 standard
- [ ] Linux arm6 Docker
- [x ] Windows amd64 standard
- [ ] Windows amd64 Docker (WSL backend)
- [ ] macOS amd64 standard
- [ ] macOS amd64 Docker
- [ ] Other (please describe)

## Describe the issue

when pushing by Android Pixel 4A5G of back camera with Chrome, it high probability dones't work.
![1 (1)](https://github.com/bluenviron/mediamtx/assets/16969250/f4d5be2f-78c3-4ad8-8380-a953dabb801d)
![1 (2)](https://github.com/bluenviron/mediamtx/assets/16969250/24fe04ce-4b7d-45a5-9929-fdf3cb287bc9)

## Describe how to replicate the issue

1. start the server
2. publish with Android Chrome 
3. read with OBS browers source

## Did you attach the server logs?

no

## Did you attach a network dump?

no

**Comment by aler9** [comment]
Hello, after doing some research it seems that that error is thrown when the camera is already in use. The server indeed requests the camera twice, the first call is performed in order to get all available codecs and devices, and the second call is performed in order to perform the streaming.

I've edited the code in order to stop the camera request before starting the second one, please test this nightly release and let me know if it works (click on artifacts, binaries):

https://github.com/bluenviron/mediamtx/actions/runs/7234099772

**Comment by iesv** [comment]
YES! It works! Thank you aler9!
however, it cannot show 1920x1080 size in chrome.

**Comment by aler9** [comment]
Thanks for providing the feedback and merry christmas to you too from Italy :)

Regarding the size issue, open another issue and provide details.

Regarding latency, the server doesn't introduce any latency, it just routes packets from an end to the other. Try to debug your streaming pipeline and to find the point in which latency is introduced. For instance, try to publish with a desktop browser and check whether latency is lower - it that is the case, it means that the problem lies in the smartphone, otherwise, the problem lies in something in the server side, and so on.
